backend/inspirehep/records/marshmallow/literature/common/author.py

```diff
@@ -14,28 +14,14 @@
 from inspirehep.records.marshmallow.utils import get_first_value_for_schema
 
 
-class AuthorSchemaV1(Schema):
+class FirstAuthorSchemaV1(Schema):
 
     recid = fields.Method("get_recid", default=missing, attribute="record")
```

Comment:
I left these because yesterday in the group meeting there was this issue with the name ```Tim``` being in emails only and we said we would search for names in emails as well in the future or something related to emails.